### DISCUSSION TITLE
Kubernestes logs with promtail and loki

### DISCUSSION THREAD

**Discussion by NiharsGIT** [siderolabs/talos#7002] [Q&A]
I have been using talos with proxmox to host my kubernetes and was wanting to use LGTM stack for observability,
I am facing an issue configuring promtail with loki on Talos cluster, mainly because I am unable to locate the log file path which is supposed to be /var/log/pods on the talos nodes, I only found these [attached image] files under /var/log/**
I have also tried /var/lib/containers/** but no luck

I have seen the logging document https://www.talos.dev/v1.3/talos-guides/configuration/logging/ which was not helpful, it only seems to help with Talos system logs, not kubernetes log

in general, I wonder how these logging tools can be used with Talos, is Talos handling kubernetes logs differently? any thoughts on how I can set these up?

![image](https://user-images.githubusercontent.com/22151004/227718349-99ca423c-ffaf-4a8c-998b-3d3eb7cd4831.png)

**Comment by smira** [comment]
In the documentation you linked, the path is in the example, it's standard `/var/log/containers` (not `lib`): https://www.talos.dev/v1.3/talos-guides/configuration/logging/#filebeat-example
